packages/form-hooks/src/NumberField/NumberField.tsx

```diff
@@ -88,6 +90,12 @@ const NumberField = ({
         if (onChange) {
           onChange(newValue);
         }
+
+        if (returnAsNumber) {
+          newValue = parseFloat(newValue);
+          if (Number.isNaN(newValue)) newValue = undefined;
```

Comment:
Burde denne settast til null i staden for undefined sånn som lenger opp? (Veit at react-hook-form håndterar desse forskjellig utan at eg huksar korleis akkurat no)

Reply:
Jeg har ikke opplevd noen forskjell i hvordan react-hook-form har håndtert `undefined` kontra `null`. Strengt tatt tenker jeg det blir riktig slik det er nå. `null` representerer en verdi som bevisst satt til "tom/blank", mens `undefined` representerer mangelen på en angitt verdi.

Reply:
Ja eg er forsovidt einig i det, men som sagt, eg er nokså sikker på at det har oppført seg ulikt tidlegare, i enkelte situasjonar iallfall. Men uansett, no ligg jo dette bak ein if, så det er berre å merga for min del

Reply:
Men du har helt rett, jeg ser at de skriver i doc at man skal bruke null og ikke undefined i react-hook-form. Det forårsaket ingen problemer da jeg testet det på forskjellige måter, men det er best å følge instruksjonene :)